src/Honeybee_Generator_PV.py

```diff
@@ -57,7 +53,7 @@
 
 ghenv.Component.Name = "Honeybee_Generator_PV"
 ghenv.Component.NickName = 'PVgen'
-ghenv.Component.Message = 'VER 0.0.58\nJAN_16_2015'
+ghenv.Component.Message = 'VER 0.0.58\nJAN_18_2015'
 ghenv.Component.Category = "Honeybee"
 ghenv.Component.SubCategory = "12 | WIP" #"06 | Honeybee"
 #compatibleHBVersion = VER 0.0.56\nFEB_01_2015
```

Comment:
You forgot to update the compatible date for all the components. This will make a huge issue when users use this component with an older version of Honeybee. You need to fix it for the both components.